commit bf2f2229bb108a7bdd39326eb4e886e2f47df960
Author: Michael Tokarev <mjt@tls.msk.ru>
Date:   Thu Jan 23 14:41:53 2014 +0100

    testsuite: fix last "which" change
    
    In commit afa63b2dcdc9b9 I replaced `type -p' with
    `command -pv'.  But actually it is wrong, the right
    substitution is `command -v'.  We need to find our
    busybox which is in the first directory in $PATH, so
    `command -p' should not be used because it uses
    default PATH, not current value of PATH where our
    busybox binary resides.
    
    Signed-off-by: Michael Tokarev <mjt@tls.msk.ru>
    Signed-off-by: Denys Vlasenko <vda.linux@googlemail.com>

diff --git a/testsuite/which/which-uses-default-path b/testsuite/which/which-uses-default-path
index 349583d..92b6018 100644
--- a/testsuite/which/which-uses-default-path
+++ b/testsuite/which/which-uses-default-path
@@ -1,4 +1,4 @@
-BUSYBOX=$(command -pv busybox)
+BUSYBOX=$(command -v busybox)
 SAVED_PATH=$PATH
 unset PATH
 $BUSYBOX which ls
